src/passes/Print.cpp

```diff
@@ -2328,11 +2355,23 @@ struct PrintSExpression : public OverriddenVisitor<PrintSExpression> {
     }
     if (curr.params.size() > 0) {
       o << maybeSpace;
-      o << ParamType(curr.params);
```

Comment:
Were these the only places in which we used `ParamType` and `ResultType`? Can we remove them?

Reply:
They are still used in event printing, which I'm not sure what to do about, but if we decide to do it there as well then we can remove them. However, they may also be useful for debugging.